fixtures/Delegated/__init__.py

```diff
@@ -2,7 +2,7 @@
 
 
 def build():
-    fixture = ConsistencyVariantFixtureBuilder('Delegated')\
+    fixture = ConsistencyVariantFixtureBuilder('Delegated', { 'use_snapshot_hashes': True })\
```

Comment:
Do we need a fixture where this is false for testing?

Reply:
Not as far as I can tell. This fixture was meant for testing delegation of trust below the targets role, which isn't affected by snapshot and timestamp hasing.